docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.7'
+services:
+  media:
+    build:
+      context: .
+      target: app
+    ports:
+      - 4001
+    environment:
+      NUM_CLUSTERS: 1
+      STORAGE_PATH: ./.media
+      TOKEN: super-secret
+    volumes:
```

Comment:
I think it would make sense to add the `./.media` folder to the volumes. Otherwise all the images will be gone if somebody does a `docker-compose down`. In the `package.json` is already a script `clean:media` that will remove the folder.